### PR TITLE
This pull request integrated into request #133

### PR DESCRIPTION
**Pull request by MasterWaifuman**
Modified:

   Nicholas Abdicates:
       Added option for player to change country to Soviet Union
   Treaty of Brest-Litovsk:
       According to real treaty German Empire puppets United Baltic Duchy and Lithuania

Added:
   Peace between Entente and Soviet Union in 1925:
       To end eternal war between Entente and Soviets (only if they at war(in 1925))

Obviosly also modifies localisation files and adds gfx for Baltic Duchy

This pull request integrated in #133

### CODE REVIEW

**Comment by DevilIris** [comment]
Nice job did you tested it?

**Comment by MasterWaifuman** [comment]
Yes, I tested it. But it hard to test trigger for event that stops eternal war between soviets and entente (mostly because it triggers only in 1925). But event itself works.

**Comment by DevilIris** [comment]
It can be stop before if Mongolia have a neutrality spirit with a tension limit factor

**Comment by MasterWaifuman** [comment]
But mustly not and it leads to war between entente and soviet union that never ends (because of game mechanics that need major countries capitulate to end war). It happens after treaty of brest-litovsk and other things.

**Comment by MasterWaifuman** [comment]
I played as German Empire and when I lost ww1 both soviets and entente couldn't end the war.

**Comment by MasterWaifuman** [comment]
irl this treaty named "Soviet–Japanese Basic Convention"

**Comment by MasterWaifuman** [comment]
And another thing is that it triggers after 1925 if soviets has war with japan and france(leader of entente). And fires only once. And you always can decline treaty

**Comment by DevilIris** [comment]
Cool i guess that @Wolferos  is going to merge this. 

If you want to merge something else is always welcome

**Comment by MasterWaifuman** [comment]
My native language is russian, so I will add translation to this event.(and may be not only for this event)

**Review by DevilIris — APPROVED** [review]

**Review by DevilIris — APPROVED** [review]

**Comment by DevilIris** [comment]
Also @MasterWaifuman  Can you continue the Soviet Union Focus tree ?  

Someone is finally working on russia so if you want to make the focus tree welcome here is thread if you want some ideas : https://github.com/Wolferos/Hearts-of-Iron-IV-The-Great-War/issues/24

**Comment by MasterWaifuman** [comment]
I think i can code @Darksaber501st focuses. Next i think would be other default focuses ( like industry, air naval and other )

**Comment by DevilIris** [comment]
yes the political and the diplomatic are unique but the other are the generic ones follow the examples like Mexico or the Netherlands

**Comment by MasterWaifuman** [comment]
I can also add reconstruction after civil war. First focus which unlocks others and idea like in vanilla Spain.

**Comment by DevilIris** [comment]
Yes but you also need to remember the dates also be aware of the post war
focuses. The URSS attack poland in 1919-1920 so we that parts . Etc.

DevilIris

**Comment by MasterWaifuman** [comment]
I created the base for focus tree. I think it's enough. I will start working on effects and other stuff.
soviet focus tree 1
soviet focus tree 2
soviet focus tree 3

**Comment by DevilIris** [comment]
Nice it looks very cool .

DevilIris

**Comment by MasterWaifuman** [comment]
Something like that i think
soviet focus tree 1
soviet focus tree 2
soviet focus tree 3

**Comment by DevilIris** [comment]
is perfect for the current version

**Comment by MasterWaifuman** [comment]
I think for the next update i will work on availability sections of SOV focuses ( some of them I made already ) and translations.

**Comment by DevilIris** [comment]
did you test out the focus ? in the view of the players and non player to make it historical?

**Comment by DevilIris** [comment]
also i am going to test this :D

**Comment by MasterWaifuman** [comment]
i looked through them

**Comment by MasterWaifuman** [comment]
i cannot completely understand phrase "in the view of the players and non player to make it historical?" ( i'm russian retard and not so good in internet english ). so can you rephrase it?

**Comment by DevilIris** [comment]
If you play with the Sov. also if you watch as any other country the Sov. AI with the focuses. As well with russia

**Comment by MasterWaifuman** [comment]
i didn't tested historical mode ( because i need to wait until ai do these focuses ). i took "historical ai on" modifier from one of vanilla soviet demands ( bessarabia demand if exactly ) but i don't know how it would work in focus tree. But it is modifier and should work anywhere where can put modifiers.

**Review by DevilIris — APPROVED** [review]
It works fine in my current game...ñ (Only tested by a player view on the nation).

**Comment by DevilIris** [comment]
@MasterWaifuman  Also can you pull request this on my repository so i can work with this on git hub pls

**Comment by MasterWaifuman** [comment]
Soooo i figured out how to send pull request not to me but to you and i did it

**Comment by DevilIris** [comment]
Nice if you want to test the version with all the changues download the zip from my develop the things that you have done and i have done it will be there. Meanwhile Wolferos merge this on the main develop

**Comment by MasterWaifuman** [comment]
In the country choice menu germany has general staff idea but when you start game you haven't that idea. Is it how it supposed to be or this is a bug. i think it is not such big problem but little confusing

**Comment by MasterWaifuman** [comment]
This pull request is combination of #125 and #133

**Comment by DevilIris** [comment]
pull request the last files to me pls . I can´t merge it in my repo.

**Comment by DevilIris** [comment]
also do you have steam ? Add me : devilemi

**Comment by MasterWaifuman** [comment]
Sorry can't find you
image

**Comment by DevilIris** [comment]
What is your nick or id? so i can find you

DevilIris

**Comment by MasterWaifuman** [comment]
My nick is: Adolf Putin ( don't ask why :).

**Comment by MasterWaifuman** [comment]
and id: 76561198115812411 i suppose

**Comment by MasterWaifuman** [comment]
I figured out that i can't add friends because i didn't spend monet on steam

**Comment by DevilIris** [comment]
i send you the request

**Comment by MasterWaifuman** [comment]
i accepted
